README.rst

```diff
@@ -23,12 +23,12 @@ Available Features
 ------------------
 
 * Displays ``Uptime``, ``Memory Status``, ``Load Status``, ``Addresses``, ``Neighbours`` of the **Device**
-* Displays ``Monitoring Graphs`` namely ``Uptime``, ``Packet Loss``, ``Round Trip Time``, ``Traffic`` in the form of lively charts
-* ``Monitoring Graphs`` can be viewed at resolutions of 1 day, 3 days, a week, a month and a year
+* Displays ``Monitoring Charts`` namely ``Uptime``, ``Packet Loss``, ``Round Trip Time``, ``Traffic`` in the form of lively charts
+* ``Monitoring Charts`` can be viewed at resolutions of 1 day, 3 days, a week, a month and a year
 * CSV Export of monitoring data
 * It allows you to add custom `Charts <https://github.com/openwisp/openwisp-monitoring/#openwisp_monitoring_charts>`_
-* It allows to create custom ``Metric``, select a ``Chart`` and set a ``Threshold``
-* Notifications are sent whenever ``Threshold`` value is crossed
+* It allows to create custom ``Metric``, select a ``Chart`` and set a ``AlertTheshold``
```

Comment:
typo: AlertThreshold

Reply:
Thanks for bringing out to my notice. I used a regex so a typo at one place means typo everywhere. It's corrected now.